pyproject.toml

```
@@ -9,7 +9,7 @@ readme = "README.rst"
 requires-python = ">=3.8"
 license = {text = "BSD License"}
 authors = [{name = "The PyGMT Developers", email = "pygmt.team@gmail.com"}]
-keywords = []
+keywords = ["cartography", "geodesy", "geophysics", "geospatial", "oceanography", "seismology"]
```

The list is a bit long, so maybe break it into multiple lines.

```suggestion
keywords = [
    "cartography",
    "geodesy",
    "geophysics",
    "geospatial",
    "oceanography",
    "seismology"
]
```